publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -493,7 +503,7 @@ public void shutdownCompleted(ShutdownSignalException cause) {
             });
             BasicProperties msgProps = MessageProperties.BASIC;
```

Comment:
Why always assign BASIC properties to msgProps and then overwrite it? I'd do

`BasicProperties msgProps = usePersistence ? PERSISTENT_BASIC_APPLICATION_JSON : MessageProperties.BASIC;`